dev-haskell/alex/alex-3.2.7.2.ebuild

```
@@ -41,6 +37,7 @@ DEPEND="${RDEPEND}
 src_prepare() {
 	if use doc; then
 		cp -a "${GIT_S}/doc/" "${S}" || die
+		PATCHES+=( "${FILESDIR}/${PN}-3.2.7.2-fix-docs.patch" )
```

Thanks for catching this! I believe it needs to be slightly different, though:

```suggestion
		eapply "${FILESDIR}/${PN}-3.2.7.2-fix-docs.patch"
```

Does your method work?

---

The patch was applied in `haskell-cabal_src_prepare`, so it worked. I updated it to `eapply` since it seems more straightforward.